packages/slice-machine/src/domain/customType.ts

```diff
@@ -425,6 +447,41 @@ export function reorderField(args: ReorderFieldArgs): CustomType {
   return newCustomType;
 }
 
+// TODO add tests
+export function updateUidField(
+  label: string,
+  customType: CustomType,
+): CustomType {
+  const field = getUidField(customType);
+  if (!field) return customType;
+
+  const section = findSectionWithUidField(customType);
+  const sectionId = section ?? "Main";
+
+  const newField = {
+    ...field,
+    config: {
+      ...field.config,
+      label,
+    },
+  };
+
+  const updatedSection = updateFields({
+    fields: customType.json[sectionId],
+    previousFieldId: "uid",
```

Comment:
❓#question

But here then I don't understand, we already saw from your previous code that there was no UID existing, how can we have a previous uid field in that case?